.github/workflows/windows_ci.yml

```diff
@@ -1,6 +1,7 @@
 name: Windows CI
 
 on:
+  workflow_dispatch:
```

Comment:
... maybe the Linux CI could also benefit from it? 🙂

Reply:
Let's do this!

Reply:
done